safetrace-app/Features/Contact Tracing/ContactTracingViewController.swift

```diff
@@ -461,6 +468,20 @@ class ContactTracingViewController: UIViewController {
         }
         return privacyAndTermsTextView
     }
+    
+    private func makeAppVersionLabel() -> UILabel {
+        let versionLabel = UILabel()
+        versionLabel.font = .smallRegular
+        versionLabel.textColor = .stGrey40
+        versionLabel.isHidden = true
+        
+        if let v = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String, let b = Bundle.main.object(forInfoDictionaryKey: "CFBundleVersion") as? String {
+            versionLabel.text = "v\(v) (\(b))"
```

Comment:
should we remove the (build number) portion? It looks a little busy and is probably not helpful for users since we'll only have one build per release versin.

Reply:
this is the same string we use on citizen. I don't think any users are going to notice this text to begin with

Reply:
Hmm fair enough. I notice it but maybe that's just because I'm a dev lol.